opennms-provision/opennms-detector-simple/src/main/java/org/opennms/netmgt/provision/detector/snmp/Win32ServiceDetector.java

```diff
@@ -63,15 +65,11 @@ public String getWin32ServiceName() {
     public void setWin32ServiceName(String serviceName) {
         m_win32SvcName = serviceName;
         LOG.debug("setWin32ServiceName: setting service name to {}", serviceName);
-        int snLength = serviceName.length();
-        
-        StringBuffer serviceOidBuf = new StringBuffer(SV_SVC_OPERATING_STATE_OID);
-        serviceOidBuf.append(".").append(Integer.toString(snLength));
-        for (byte thisByte : serviceName.getBytes()) {
```

Comment:
I was given a copy of an snmpwalk from a non-us host which had unicode values in the index strings. I tried using @JunitSnmpAgent on the a test method, but that didn't work, so I created a new unit test which uses this new snmpwalk output.

I also uncovered a bug in my original version. I needed to get the length in bytes, not the character length since unicode characters count as 1 character, but can be represented by multiple bytes.